cms/templates/visibility_editor.html

```diff
@@ -27,8 +27,8 @@ <h4 class="title">${_('You have not set up any groups to manage visibility with.
         </div>
     % elif is_staff_locked:
         <div class="summary-message summary-message-warning visibility-summary-message">
-            <i class="icon icon-warning-sign"></i>
-            <p class="copy">${_('The Unit this component is contained in is hidden from students. Visibility settings here will be trumped by this.')}</p>
+            <i class="icon fa fa-exclamation-triangle" aria-hidden="true"></i>
+            <p class="copy"><span class='sr'>${_('Warning: ')}</span>${_('The Unit this component is contained in is hidden from students. Visibility settings here will be trumped by this.')}</p>
```

Comment:
Some notes to the translators would be good here. It is important that "Warning:" have a trailing space.

A better way to do this would be with placeholders to keep the HTML out of the translatable string, but allow the translators to deal with the string as a whole. For example,
"{span_tag_start}Warning: {span_tag_end} The Unit this component...".format